src/redux/sagas/motions/createEditDomainMotion.ts

```diff
@@ -35,10 +33,172 @@ import {
   uploadAnnotation,
   initiateTransaction,
   createActionMetadataInDB,
+  getPermissionProofsLocal,
+  getChildIndexLocal,
 } from '../utils/index.ts';
 
 // import { ipfsUpload } from '../ipfs';
 
+type CreateTransactionPayload = {
```

Comment:
It was difficult to see that they're identical after all the revisions and fatigue 😭  It makes sense to use the same type. Thanks for spotting this @bassgeta !

Reply:
After looking at it again, there a couple of differences here actually. You've got these ones that are not relevant to the function:
```js
colonyName,
domainName,
domainColor,
domainPurpose,
```
I think we can leave this as-is to avoid too much `Pick` | `Omit` confusions 🤔

Reply:
I would vote though for using the `Omit` as it makes it easier for possible future updates on the `MotionDomainCreateEditPayload` or extend the `MotionDomainCreateEditPayload` to include the extra fields 🙈